docs-gen/content/rule_set/basics.md

```diff
@@ -109,17 +111,25 @@ The VSS specification must adhere to YAML syntax. To keep the standardized VSS s
 
 ### Naming Conventions
 
-The recommended naming convention for node elements is to use camel case notation starting with a capital letter. It is recommended to use only
-`A-Z`, `a-z` and `0-9` in node names. For boolean signals it is recommended to start the name with `Is`.
+Node names in VSS must follow the restrictions specified in [Yaml](https://yaml.org/spec/1.2.2/#chapter-5-character-productions) i.e. must only use the printable subset of the Unicode character set.
+
+Tools and Generators supporting VSS may however put additional restrictions on which characters that are allowed. For maximum portability, node names in the VSS standard catalog must fulfill the following rules
+
+* Node names in the VSS standard catalog shall use camel case notation starting with a capital letter.
+* Node names in the VSS standard catalog shall use only `A-Z`, `a-z` and `0-9` in node names.
+* Boolean signals must start with `Is`.
+* Full path of node names must be unique also when using case insensitive comparison.
+  It is for example not allowed to have both `Vehicle.Abc` and `Vehicle.ABC` in the standard catalog
+  even if that is valid VSS as VSS is case sensitive.
 
 Examples:
 
 ```
-SomeBranch.AnotherBranch.MySignalName
 Vehicle.Cabin.Door.Row1.Left.IsLocked
+SomeOtherTree.AnotherBranch.MySignalName
```

Comment:
Not sure for what is this an example, doesn't it state above every branch need to start with "Vehicle"?

Reply:
In the VSS standard catalog the root name is `Vehicle`, but you may have a different tree using VSS syntax and then you can select whatever root name you want. Like if you use HIM to specify that multiple trees shall be used. I tried to make it more visible by renaming it from `SomeBranch` to `SomeOtherTree`. But that is sort of a special case so I am considering just removing this line.